src/inverseRobot.js

```diff
@@ -24,7 +24,20 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inversed = {};
+  const keys = Object.keys(robot);
+  const values = Object.values(robot);
```

Comment:
Read only using keys, don't rely on elements order in two independent arrays (`keys` and `values`).

```suggestion
  const keys = Object.keys(robot);
```